code/datums/objectives/objective.dm

```diff
@@ -0,0 +1,16 @@
+/datum/objective
+	var/datum/mind/owner = null //Is the objective just yours?
+	var/datum/faction/faction = null // Is the objective faction-wide?
+	var/explanation_text = "Just be yourself." //What that person is supposed to do.
+	var/is_void = 0 // Universe is doomed what's the point.
```

Comment:
Wouldn't it be better to check for the universe state, rather than presumably the universe state switching all objectives to is_void = 1?